es6Classes.js

```diff
@@ -0,0 +1,142 @@
+// Exercise 4.1 CLASSES //
+
+// var Person = function(name) {
+//     this.name = name;
+//     this.canTalk = true;
+//     this.greet = function() {
+//         if (this.canTalk) {
+//             console.log('Hello, I ' + this.name);
+//         }
+//     };
+// };
+
+// var Employee = function(name, title) {
+//     this.name = name;
+//     this.title = title;
+//     this.greet = function() {
+//         if (this.canTalk) {
+//             console.log('Hello, I ' + this.name + ', ' + this.title);
+//         }
+//     };
+// };
+// Employee.prototype = new Person();
+
+// var Customer = function(name) {
+//     this.name = name;
+// };
+// Customer.prototype = new Person();
+
+// var Mime = function(name) {
+//     this.name = name;
+//     this.canTalk = false;
+// };
+// Mime.prototype = new Person();
+
+class Person {
+	constructor(name) {
+		this.name = name;
+		this.canTalk = true;
+	}
+	greet() {
+		if (this.canTalk == true) {
+			console.log('Hello, I ' + this.name);
+		}
+	}
+}
+
+class Employee extends Person {
+	constructor(name, title) {
+		super(name);
+		this.title = title;
+	}
+	greet() {
+		if (this.canTalk == true) {
+			console.log('Hello, I ' + this.name + ', ' + this.title);
+		}
+	}
+}
+
+class Customer extends Person {
+	constructor(name) {
+		super(name);
+	}
+}
+
+class Mime extends Person {
+	constructor(name) {
+		super(name);
+		this.canTalk = false;
+	}
+}
+
+const newCustomer = new Customer('Dzmitry');
+const newEmployee = new Employee('Sergey', 'Manager');
+const newMime = new Mime('Anton');
+// console.log(newCustomer);
+// console.log(newEmployee);
+// console.log(newMime);
+
+// Exercise 4.2 CALCULATOR//
+
+class Calculator {
+	constructor() {
+		this._history = [];
+	}
+	_addingToHistory(val) {
```

Comment:
try to use [private modificatory](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Classes/Private_class_fields)